### PR TITLE
Add flite 3ds portlib

### PR DESCRIPTION
**Pull request by Pirater12**
I was not sure about what to put in pkgver, so it has `master` for now

### CODE REVIEW

File: 3ds/flite/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
+pkgname=3ds-flite
+pkgver=master
+pkgrel=1
+pkgdesc='A small fast portable speech synthesis system'
+arch=('any')
+url='https://github.com/festvox/flite'
```
Comment (WinterMute) [inline 3ds/flite/PKGBUILD]: This should be http://festvox.org/flite/

File: 3ds/flite/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,40 @@
+pkgname=3ds-flite
+pkgver=master
```
Comment (WinterMute) [inline 3ds/flite/PKGBUILD]: This should be 2.1

File: 3ds/flite/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+pkgrel=1
+pkgdesc='A small fast portable speech synthesis system'
+arch=('any')
+url='https://github.com/festvox/flite'
+license=('MIT')
+options=(!strip libtool staticlibs)
+depends=()
+makedepends=('3ds-pkg-config' 'devkitpro-pkgbuild-helpers')
+
+source=("https://github.com/festvox/flite/archive/$pkgver.zip")
```
Comment (WinterMute) [inline 3ds/flite/PKGBUILD]: Should be http://festvox.org/flite/packed/flite-${pkgver}/flite-${pkgver}-release.tar.bz2

Comment (Pirater12) [reply 3ds/flite/PKGBUILD]: WinterMute wouldn't it be better to keep up with the github repository?

File: 3ds/flite/PKGBUILD [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+url='http://festvox.org/flite/'
 license=('MIT')
 options=(!strip libtool staticlibs)
 depends=()
 makedepends=('3ds-pkg-config' 'devkitpro-pkgbuild-helpers')
 
-source=("https://github.com/festvox/flite/archive/$pkgver.zip")
+source=("http://festvox.org/flite/packed/flite-${pkgver}/flite-${pkgver}-release.tar.bz2")
 
 sha256sums=('2d48f3cc733d24d358f53fa1719ffa0be40b8b55214da634566e7fba2f2bff95')
```
Comment (WinterMute) [inline 3ds/flite/PKGBUILD]: This will need updated for the new archive, won't it?

Comment (Pirater12) [reply 3ds/flite/PKGBUILD]: It needs to be updated, sorry!

**Pull request merged**